aerospike/metadata.csv

```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```

Comment:
@whyrg I think all we are missing is the metric metadata. It's probably the most boring part so no wonder we almost forgot about it! 😅 

You can look at some of the other integrations for guidance:
- metric_name (required): the name of the metric, plain and simple (eg. `aerospike.foo.bar`)
- metric_type (required): (note that as confusing as it sounds metrics we submit as rates should actually be listed as gauges in the CSV metric list, rates are a different kind of metric - TLDR; just set `gauge` for `rates` and `gauges`)  
   + gauge
   + counter
   + rate
   + count
- interval: (optional) The interval used for conversion between rates and counts. Since we don't have actual rates (remember what I mentioned above about rates being gauges?), you can probably leave this empty.
- unit_name (optional - but add if possible): the name of the units for the metrics.
- per_unit_name (optional): the name of the unit per type. For instance `blocks/second` would have `unit_name` `block`, and `per_unit_name`: `second`.
- description: (optional - but add if at all possible) A description of the metric. Limit of 400 characters. 
- orientation (required):
  + -1 - smaller values are better
  + 0 - no values are better or worse
  + 1 - larger values are better
- integration (required): this value will be `aerospike`
- short name (optional): a shortened version of the metric name.

If you can fill those in, I will then go over it and amend whatever requires fixing, and merge! 

Thanks for your help and patience @whyrg

Reply:
You can find some more details in the website: https://docs.datadoghq.com/developers/integrations/integration_sdk/#metadata-csv

Reply:
Sure thing. We have a lot of metrics so this will take a while.